src/index.html

```diff
@@ -12,7 +12,48 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="product-card__image"
+        alt="imac"
+      />
+
+      <div class="product-card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product-card__code">Product code: 195434</div>
+
+      <div class="stars">
+        <div class="stars-4">
+          <div class="stars-4__star"></div>
+          <div class="stars-4__star"></div>
+          <div class="stars-4__star"></div>
+          <div class="stars-4__star"></div>
+          <div class="stars-4__star"></div>
+        </div>
+        <span class="reviews">Reviews: 5</span>
```

Comment:
It seems like the `stars` block is intended to be a rating component. However, the class naming suggests that it is tied to a specific rating ('stars-4'). Consider using a more general class name and control the number of active stars with a modifier class or through a CSS class that represents the rating dynamically.